src/lib.rs

```diff
@@ -273,33 +273,51 @@ pub mod auth {
         })
     }
 
+    fn can_log_audit() -> bool {
+        Spi::get_one::<i64>("SELECT 1 FROM pg_extension where extname = 'pgaudit'")
```

Comment:
this is debatable. I'm not sure what's the optimal way of checking it.

I think it can't be reliably cached. So with current implementation it will be executed on each function call

Reply:
it was suggested to go with dedicated GUC and use it per analogy to https://github.com/neondatabase/neon/pull/11500/files